src/Server.php

```diff
@@ -0,0 +1,91 @@
+<?php
+
+namespace React\Http;
+
+use React\Http\Middleware\LimitConcurrentRequestsMiddleware;
+use React\Http\Middleware\RequestBodyBufferMiddleware;
+use React\Http\Middleware\RequestBodyParserMiddleware;
+use React\Socket\ServerInterface;
+
+/**
+ * Facade around StreamingServer with sane defaults for 80% of the use cases.
+ * The requests passed to your callable are fully compatible with PSR-7 because
+ * the body of the requests are fully buffered and parsed, unlike StreamingServer
+ * where the body is a raw ReactPHP stream.
+ *
+ * Wraps StreamingServer with the following middleware:
+ * - LimitConcurrentRequestsMiddleware
+ * - RequestBodyBufferMiddleware
+ * - RequestBodyParserMiddleware
```

Comment:
What about the `enable_post_data_reading` ini setting as documented for `RequestBodyParserMiddleware`? Arguably, we *may* want to remove the `RequestBodyParserMiddleware` from the middleware list in this case to mimic PHP's default behavior?

Reply:
Updated the PR to respect `enable_post_data_reading`